### ISSUE TITLE
Top row not only still acts as a special row, but the keys are now inaccurate

### ISSUE THREAD

**Issue by childli** [WeirdTreeThing/cros-keyboard-map#36]
I'm on an Acer 516 GE running Nobara.

Basically the title. Since my keyboard doesn't have a right arrow, though the script apparently expects one, my refresh is the right arrow, maximize is refresh, etc.

None of them act like a function key.

How can I uninstall this script? I give up on it.

**Comment by WeirdTreeThing** [comment]
Interesting

**Comment by WeirdTreeThing** [comment]
The script is supposed to detect the layout of your top row, so I don't see how it would be inaccurate. And having it suddenly work makes zero sense.

**Comment by childli** [comment]
Yeah, it's strange. Back button works as a back button, but refresh acts as a next key, maximize is refresh, the weird box-with-three-lines key acts as the screenshot key, the actual screenshot key acts as the super key, and then from there, decrease/increase brightness and mute/decrease/increase volume all work as normal. Any ideas? Or should I continue with uninstall/key reversion if possible?

**Comment by WeirdTreeThing** [comment]
Whats the output of `./cros-keyboard-map.py`

**Comment by childli** [comment]
Hmm, I just ran the script again, this time telling it not to invert. It seemed to work properly now and they keys do what they're designed to - but function keys still are broke. I'm 90% confident it's a weird software issue and not your fault or mine. Sorry for wasting your time and thank you for your time.

**Comment by WeirdTreeThing** [comment]
Don't worry, I want to figure out what's wrong with this. Can you send the generated `cros.conf` file in the cloned script directory?
